sceptre/scipool/templates/prod/lambda-budgets/budget-rules.yaml

```diff
@@ -1,9 +1,15 @@
 # budget rules per user by synapse team id
 teams:
-  '3469335': # Schwannomatosis Open Research Collaborative Service Catalog Users
+  - '3469335': # Schwannomatosis Open Research Collaborative Service Catalog Users
```

Comment:
I think this should be a dict with a new key, not converted to a list. https://github.com/Sage-Bionetworks-IT/lambda-budgets/blob/master/budget/config.py#L9

Reply:
Good catch, thanks.